easybuild/easyconfigs/n/NAG/NAG-26-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'Binary'
+
+name = 'NAG'
+version = '26'
+
+homepage = 'http://www.nag.co.uk'
+description = """The worlds largest collection of robust, documented, tested and maintained numerical algorithms."""
+
+toolchain = {'name': 'GCC', 'version': '6.4.0-2.28'}
+
+nagname = 'fll6i%sdfl' % version
+
+source_urls = ['https://www.nag.co.uk/downloads/impl']
+sources = [nagname + '.tgz']
+checksums = ['9725e843d1f9b1e509cae0bc2b509cd24192b2e347aa7b1a7792b8b9efb5863d']
+
+preinstallopts = 'tar -xvf %s.tgz && ' % nagname
```

Comment:
This line can be dropped when using `PackedBinary` (you'll need to change the `+=` below to just `=` too of course).